src/schema/v1.js

```diff
@@ -0,0 +1 @@
Path+"/dataschema",schemaPath:"#/definitions/dataschemadef/format",keyword:"format",params:{format: "uri"},message:"must match format \""+"uri"+"\""}];return false;}}}}}var valid0 = _errs16 === errors;}else {var valid0 = true;}if(valid0){if(data.subject !== undefined){let data6 = data.subject;const _errs19 = errors;const _errs20 = errors;if((typeof data6 !== "string") && (data6 !== null)){validate20.errors = [{instancePath:instancePath+"/subject",schemaPath:"#/definitions/subjectdef/type",keyword:"type",params:{type: schema29.type},message:"must be string,null"}];return false;}if(errors === _errs20){if(typeof data6 === "string"){if(func8(data6) < 1){validate20.errors = [{instancePath:instancePath+"/subject",schemaPath:"#/definitions/subjectdef/minLength",keyword:"minLength",params:{limit: 1},message:"must NOT have fewer than 1 characters"}];return false;}}}var valid0 = _errs19 === errors;}else {var valid0 = true;}if(valid0){if(data.time !== undefined){let data7 = data.time;const _errs22 = errors;const _errs23 = errors;if((typeof data7 !== "string") && (data7 !== null)){validate20.errors = [{instancePath:instancePath+"/time",schemaPath:"#/definitions/timedef/type",keyword:"type",params:{type: schema30.type},message:"must be string,null"}];return false;}if(errors === _errs23){if(errors === _errs23){if(typeof data7 === "string"){if(func8(data7) < 1){validate20.errors = [{instancePath:instancePath+"/time",schemaPath:"#/definitions/timedef/minLength",keyword:"minLength",params:{limit: 1},message:"must NOT have fewer than 1 characters"}];return false;}else {if(!(formats4.validate(data7))){validate20.errors = [{instancePath:instancePath+"/time",schemaPath:"#/definitions/timedef/format",keyword:"format",params:{format: "date-time"},message:"must match format \""+"date-time"+"\""}];return false;}}}}}var valid0 = _errs22 === errors;}else {var valid0 = true;}if(valid0){if(data.data !== undefined){let data8 = data.data;const _errs25 = errors;if((((typeof data8 != "object") && (typeof data8 !== "string")) && (!((typeof data8 == "number") && (isFinite(data8))))) && (typeof data8 !== "boolean")){validate20.errors = [{instancePath:instancePath+"/data",schemaPath:"#/definitions/datadef/type",keyword:"type",params:{type: schema31.type},message:"must be object,string,number,array,boolean,null"}];return false;}var valid0 = _errs25 === errors;}else {var valid0 = true;}if(valid0){if(data.data_base64 !== undefined){let data9 = data.data_base64;const _errs28 = errors;if((typeof data9 !== "string") && (data9 !== null)){validate20.errors = [{instancePath:instancePath+"/data_base64",schemaPath:"#/definitions/data_base64def/type",keyword:"type",params:{type: schema32.type},message:"must be string,null"}];return false;}var valid0 = _errs28 === errors;}else {var valid0 = true;}}}}}}}}}}}}else {validate20.errors = [{instancePath,schemaPath:"#/type",keyword:"type",params:{type: "object"},message:"must be object"}];return false;}}validate20.errors = vErrors;return errors === 0;}
```

Comment:
I don't think we should check in compiled JS like this in our source repo.

Reply:
I understand your sentiment here and I considered not adding it to the PR. In most cases, you of course do not want to add generated source to revision control. However, there are some cases where it might make sense. For example, `package-lock.json` is a generated file, and it's considered best practice to include it in your repository. Many golang projects also include generated code in a repo. I think it's fair to say that not doing so is a guideline, and not a hard and fast rule.

In this case, I think it perhaps _does_ make sense to include it. The main reason is that it hardly ever changes. If we were using `Makefile`, the code would only ever be generated when `schema/cloudevent.json` is generated. Of course we are not using `Makefile` so it's always generated, but still... I like the fact that we will know that it has changed, since any change appears in git. It is also nice to be able to validate the change when necessary. If `schema/cloudevent.json` changes, then `schema/v1.js` MUST also change. We will know for sure that is the case, if and only if it's checked in.

I am willing to concede, but I definitely thought about it, and definitely made the conscious choice to include it for those reasons.